lib/matplotlib/patches.py

```diff
@@ -4256,67 +4277,23 @@ def draw(self, renderer):
         if not self.get_visible():
             return
 
-        renderer.open_group('patch', self.get_gid())
-        gc = renderer.new_gc()
-
-        gc.set_foreground(self._edgecolor, isRGBA=True)
-
-        lw = self._linewidth
-        if self._edgecolor[3] == 0:
-            lw = 0
-        gc.set_linewidth(lw)
-        gc.set_dashes(self._dashoffset, self._dashes)
-
-        gc.set_antialiased(self._antialiased)
-        self._set_gc_clip(gc)
-        gc.set_capstyle('round')
```

Comment:
The context uses `gc.set_capstyle(self._capstyle)` instead. This is 'butt' by default (inherited from `Patch`). It's reasonable to draw the value from `self._capstyle`. Should the default for this value be 'round' to have backward compatibility or do you want to change it intentionally to be the same default as for `Patch`?

Reply:
I forced it to "round" above (in the setattr_cm).  Yes, ignoring the capstyle was probably a long-standing bug (the kind of things that such refactors uncover...) but changing that would result in changes in the baseline images (... that's how I realized the discrepancy).  I'll leave this as it is until someone actually complains or we have some othe reason to change the baseline images.